tests/tr1/tests/cstdio/test.cpp

```diff
@@ -126,13 +128,15 @@ void test_cpp() { // test C++ header
     { // test character I/O
         const char* tmpbuff;
         char tname[L_tmpnam];
-        char tmpbuf[L_tmpnam];

```

Comment:
Pre-existing: `tmpbuf` and `tmpbuff` in the same scope, really? (No change requested.)

Reply:
Yeah, this is extraordinarily and unnecessarily confusing. Now that both are `const char *`, I'll rename them.

Reply:
Actually, they can be outright eliminated.